### PR TITLE
Module6(Linq). Task

### PR DESCRIPTION
**Pull request by Nastya-Korsak**
Module 6(Linq). Task

### CODE REVIEW

**Review by alexander-trishin — APPROVED** [review]
Hi Nastya :)

Nice work! I have only 1 minor remark regarding performance.

File: Module_6/LINQ/Task1/LinqTask.cs [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+            {
+                throw new ArgumentNullException(nameof(customers));
+            }
+
+            return customers.GroupBy(
+                customer => customer.City,
+                (city, customers) =>
+                    (
+                        city,
+                        (int)Math.Round(customers.Sum(customer => customer.Orders.Sum(o => o.Total)) / customers.Count()),
```
Comment (alexander-trishin) [inline Module_6/LINQ/Task1/LinqTask.cs:173]: JFYI: The ".Average()" method from LINQ might be useful here.

Comment (Nastya-Korsak) [reply Module_6/LINQ/Task1/LinqTask.cs]: Resolve

File: Module_6/LINQ/Task1/LinqTask.cs [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        public static IEnumerable<(Customer customer, DateTime dateOfEntry)> Linq4(
+            IEnumerable<Customer> customers)
+        {
+            if (customers is null)
+            {
+                throw new ArgumentNullException(nameof(customers));
+            }
+
+            return customers
+                .Where(customer => customer.Orders.Count() > 0)
```
[the remark this answers is not in the document]
Comment (Nastya-Korsak) [reply Module_6/LINQ/Task1/LinqTask.cs]: Fixed

**Pull request merged**